lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service.rb

```diff
@@ -11,8 +11,10 @@ class MiqAeService
 
     attr_accessor :logger
 
+    # rubocop:disable ClassVars
```

Comment:
I would prefer we keep this warning enabled.  We can work on changing this to use a class instance variable in a separate PR if needed.

Reply:
Got it, thanks for pointing this out! I thought if we changed it into '@', it would be different for each instance. It is actually will be shared between instances. Updated

Reply:
Agreed, I was thinking we might be able to resolve this but looks like that would definitely change the existing logic.  I don't think we need to hold up this PR for that change.